packages/format-message-interpret/index.js

```diff
@@ -138,6 +138,21 @@ function interpretSelect (locale, id, children) {
 
 function interpretSimple (id) {
   return function format (args) {
-    return '' + args[id]
+    return '' + getArg(id, args)
+  }
+}
+
+function getArg (id, args) {
+  if (args[id] !== undefined) return args[id]
```

Comment:
Would the `in` operator be better here? `if (id in args)`, or how should `null`s be handled?

What should the following do?
```
__('{ x.y }', { 'x.y': null, x: { y: 'y' } })
```